app/controllers/groups_controller.rb

```diff
@@ -650,6 +647,18 @@ def test_email_settings
 
   private
 
+  def add_user_to_group(group, user, notify = false)
+    begin
```

Comment:
Mostly aesthetic but the `begin` and `end` are not necessary here. A function can have `rescue` without them.